token/cli/src/command.rs

```diff
@@ -3484,6 +3485,45 @@ pub async fn process_command<'a>(
                 )
             });
 
+            let transfer_fee_basis_point = arg_matches.values_of("transfer_fee_basis_points").map(|mut v| {
+                v.next()
+                    .unwrap()
+                    .parse::<u16>()
+                    .unwrap_or_else(print_error_and_exit)
+            });
```

Comment:
Let's use `value_of` for simplicity instead
```suggestion
            let transfer_fee_basis_point = value_of::<u16>(arg_matches, "transfer_fee_basis_points");
```